app/controllers/kits_controller.rb

```diff
@@ -1,6 +1,6 @@
 class KitsController < ApplicationController
   def index
-    @kits = current_organization.kits.includes(line_items: :item).class_filter(filter_params)
+    @kits = current_organization.kits.includes(:item, line_items: :item).class_filter(filter_params)
```

Comment:
This `:item` refers to the kit items themselves, while the other `:item` refers to the items inside the kits.